backend/src/main/java/gov/cdc/usds/simplereport/utils/BulkUploadResultsToFhir.java

```diff
@@ -547,14 +632,4 @@ private DiagnosticReport.DiagnosticReportStatus mapTestResultStatusToFhirValue(S
         return DiagnosticReport.DiagnosticReportStatus.FINAL;
     }
   }
-
```

Comment:
pulled this out into a utility class [here](https://github.com/CDCgov/prime-simplereport/pull/6698/files#diff-59ccc96b9db4b64105cc05d8a1fc5f9992e74d2eab062cb3d1fb4312953e4e2a)